src/scripts/main.js

```diff
@@ -1,5 +1,38 @@
 'use strict';
 
+const spiderElement = document.querySelector('.spider');
+const wallElement = document.querySelector('.wall');
+
 document.addEventListener('click', (e) => {
   // write code here
+
+  if (!wallElement.contains(e.target)) {
+    return;
+  }
```

Comment:
This condition checks if the click event occurred outside the wall element and returns early if so. This is correct as it prevents unnecessary calculations and movements when the click is outside the wall.